### ISSUE TITLE
Couple Design Thinking & Lean for your MVP

### ISSUE THREAD

**Issue by maoyang** [softnshare/productmang#1]
https://medium.com/lean-product-design/couple-design-thinking-lean-for-your-mvp-80478b8fb9f5#.z43bg5sbv

大家對這篇文章有什麼看法, 歡迎來討論

**Comment by shiningjason** [comment]
Agree! 認為 Lean Startup / MVP 也必須以 Design Thinking / Human Centered Design 的精神為基礎，莫忘初衷，我們的產品和服務就是要洞察並解決使用者最根本的問題！
